cniovs/cniovs/ovsctrl.go

```diff
@@ -0,0 +1,46 @@
+package cniovs
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func execCommand(cmd string, args []string) ([]byte, error) {
+	return exec.Command(cmd, args...).Output()
+}
+
+/*
+Functions to control OVS by using the ovs-vsctl cmdline client.
+*/
+
+func createVhostPort(sock_dir string, sock_name string) (string, error) {
+	// Create socket
+	cmd := "ovs-vsctl"
+	args := []string{"add-port", "br0", sock_name, "--", "set", "Interface", sock_name, "type=dpdkvhostuser"}
+	if _, err := execCommand(cmd, args); err != nil {
+		return "", err
+	}
+
+	// Move socket to defined dir for easier mounting
+	return sock_name, os.Rename(
```

Comment:
Should we use `os.Symlink(...)` instead of `os.Rename(...)` here?

Reply:
I saw your slack message on Friday, but just realized that this PR was here. Is it ready to merge or do you plan to update to use os.Symlink()? I don't have a preference at this point.

If you are making a change anyway, lines 150-154 in Makefile need to be removed. They installed the python script on the box when 'make install' is executed. If not, I can make the change next time I merge.

Billy

Reply:
I added the makefile edits to the PR now, thanks for pointing those out.

I don't have preference on symlink vs. move either, added this note based on the comments in slack.